milestones/3_human_brain_mapping_journal_submission/experiments/ex_2_pli_generation_optimization/example_pli_analysis.sl

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -l
+#SBATCH --job-name=example-pli-analysis
+#SBATCH --account=def-sblain # adjust this to match the accounting group you are using to submit jobs
+#SBATCH --time=0-0:45:00        # adjust this to match the walltime of your job (D-HH:MM:SS)
+#SBATCH --nodes=1     
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=40      # adjust this if you are using parallel commands
+#SBATCH --mem=90000         # adjust this according to the memory requirement per node you need (this is MegaByte)
+#SBATCH --mail-user=yacine.mahdid@mail.mcgill.ca 
+#SBATCH --mail-type=ALL
+
+# Choose a version of MATLAB by loading a module:
+module load matlab/2018a
```

Comment:
you mentioned before that you're using matlab 2020a and that the parallel computing doesn't work well without it. is that the case for this script too?

Reply:
No this is using MATLAB2018, the parallele computing engine is used to connect from your laptop directly to a cluster and sending jobs directly from MATLAB. It kinda suck so you are better off using slurm file and launching your job by using the shell to connect to beluga.